### ISSUE TITLE
Forms: empty_value in database

### ISSUE THREAD

**Issue by escopecz** [mautic/mautic#8494]
Form fields can have the `empty_value` parameter stored in the database. See this row:

```sql
INSERT INTO `form_fields` (`id`, `form_id`, `label`, `show_label`, `alias`, `type`, `is_custom`, `custom_parameters`, `default_value`, `is_required`, `validation_message`, `help_message`, `field_order`, `properties`, `label_attr`, `input_attr`, `container_attr`, `lead_field`, `save_result`, `is_auto_fill`, `show_when_value_exists`, `show_after_x_submissions`, `validation`)
VALUES
	(207, 199, 'Country', 1, 'country', 'country', 0, 'a:0:{}', NULL, 0, NULL, NULL, 6, 'a:2:{s:11:\"empty_value\";N;s:8:\"multiple\";i:0;}', NULL, NULL, NULL, 'country', 1, 0, 1, NULL, NULL);
```
This form field parameter was removed from Symfony 3 and replaced with `placeholder`.

Warning it produces on form preview:
```php
Symfony\Component\Debug\Exception\ContextErrorException: PHP Notice - Undefined index: placeholder
/app/bundles/FormBundle/Views/Field/select.html.php:35
```

**Comment by hluchas** [comment]
I'm not able to reproduce this. 

1. I created form with one select and Empty Value defined. Form field  has name `placeholder`
2. I displayed preview of form:

![Snímek obrazovky 2020-03-02 v 15 36 07](https://user-images.githubusercontent.com/12815758/75685863-918f5f00-5c9b-11ea-84f3-0072a3027325.png)

In `select.html.php:35` `$properties['placeholder']` contains this:
![Snímek obrazovky 2020-03-02 v 14 43 50](https://user-images.githubusercontent.com/12815758/75685302-a91a1800-5c9a-11ea-9a8b-0735f174ecc0.png)

because value is saved in mautic_form_fields.properties
```
a:4:{s:11:"placeholder";s:11:"Empty Value";s:8:"syncList";i:0;s:4:"list";a:1:{s:4:"list";a:2:{i:0;a:2:{s:5:"label";s:2:"Mr";s:5:"value";s:2:"Mr";}i:1;a:2:{s:5:"label";s:2:"Ms";s:5:"value";s:2:"Ms";}}}s:8:"multiple";i:0;}
```
we need 

1. batch migration for this - rename `empty_value` to `placeholder` in `mautic_form_fields.properties`
2. rename `Empty Value` translation string to `Placeholder`
3. rename translation key `mautic.form.field.form.emptyvalue` to `mautic.form.field.form.placeholder`

But i found notice somewhere else:
```
[2020-03-02 14:20:05] mautic.ERROR: Symfony\Component\Debug\Exception\ContextErrorException: PHP Notice - Undefined variable: formName - in file /Users/lukas.drahy/dev/community-fork/app/bundles/FormBundle/Views/Builder/fieldwrapper.html.php - at line 35  
``
fixed
